rfc/rfc-0019-rfc-process.md

```diff
@@ -61,7 +61,7 @@ Once a pull request is opened, the RFC is now in development and the following w
 - The team will discuss as much as possible in the RFC pull request directly. Any outside discussion will be summarized in the comment thread.
 - When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). This is step taken when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, a super majority of the team must sign off.
 - The FCP will last 7 days. If there's unanimous agreement among the team the FCP can close early.
-- For voting, the binding votes are comprised of the maintainer team. Acceptance requires a super majority of binding votes in favor. The voting options are the following: Affirmative, Negative, and Abstain. Non-binding votes are of course welcome. Super majority means 2/3 or greater of binding votes. (Abstentions are not counted in the calculation)
+- For voting, the binding votes are comprised of the maintainer team and steering committee (to be created, initially consisting of @sclevine, @rawlingsj, and @scothis). Acceptance requires a super majority of maintainer votes and at least one steering committee vote in favor. The voting options are the following: Affirmative, Negative, and Abstain. Non-binding votes are of course welcome. Super majority means 2/3 or greater of binding votes. (Abstentions are not counted in the calculation)
```

Comment:
RFC's should be timeless. It's better to create a team with the individuals listed and reference that team from the RFC.